app/models/manager_refresh/inventory/core.rb

```diff
@@ -0,0 +1,41 @@
+module ManagerRefresh::Inventory::Core
```

Comment:
> What is the benefit of having everything in Core and also in their respective managers? 

`Core` or `Platform` defines which CIs are available. A manager defines whats avail for that manager, which fields are used and which association is used.

I was thinking, we control whats going into core and then in the manager restrict it further.

But yeah, maybe this is overkill and just having it inside the `Manager` is enough